forum/templates/error.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layout.html' %}
```

Comment:
Error file allows for very basic error handling with:

return render_template("error.html", error="Error text")